README.md

```diff
@@ -218,7 +218,25 @@ Opens the user interface to set up credit cards for Apple Pay.
 
 #### `deviceSupportsApplePay() -> Promise`
 
-Indicates whether or not the device supports Apple Pay. Returns a `Boolean` value.
+Returns whether the user can make Apple Pay payments.
+User may not be able to make payments for a variety of reasons. For example, this functionality may not be supported by their hardware, or it may be restricted by parental controls.
+Returns `YES` if the device supports making payments; otherwise, `NO`.
```

Comment:
I think in JS more convenient way is return `true` or `false` instead of Obj-c `YES` or `NO`